data.sql

```diff
@@ -8,3 +8,15 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', DATE '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', DATE '2017-05-12', 5, TRUE, 11);
+
+-- Second round of insertions
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/206032336-1fd6864c-9fa5-4133-ab2b-f890dbce8114.png)

- [ ] You are using older version of your readme file. Kindly use the newer version for your microverse readme template :blush: You can get it [here](https://github.com/microverseinc/readme-template/blob/master/README.md) :+1: